### ISSUE TITLE
Cannot unfold "advanced" sections of search filters with keyboard only

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7283]
The folding sections do not get focus when pressing "tab"
![image](https://github.com/dart-lang/pub-dev/assets/8613953/37f5cbe0-5b05-479f-afae-b58fdb9abe00)
